build/pmm-client-docker/entrypoint.py

```diff
@@ -1,3 +1,59 @@
 #!/usr/bin/env python
+'''Basic Kubernetes-compatible PMM Client Docker entrypoint.'''
 
-print("PMM CLIENT 2.X DOCKER")
+from __future__ import print_function, unicode_literals
+from distutils.util import strtobool
+from os import environ, execlp
+from subprocess import Popen
+import sys
+
+
+# if PMM_AGENT_SETUP is True run:
+# `pmm-agent setup --config-file=/usr/local/percona/pmm2/config/pmm-agent.yaml`
+PMM_AGENT_SETUP = False
+
+PMM_AGENT_CONFIG = '/usr/local/percona/pmm2/config/pmm-agent.yaml'
+
+
+def pmm_agent_setup():
+    '''Setup pmm-agent.'''
+    cmd = ('pmm-agent', 'setup', '--config-file=%s' % (PMM_AGENT_CONFIG,))
+    proc = Popen(cmd, universal_newlines=True)
+    pmm_agent_stdout, pmm_agent_stderr = proc.communicate()
+    if pmm_agent_stdout is not None:
+        sys.stdout.write(pmm_agent_stdout)
+        sys.stdout.flush()
+
+    if pmm_agent_stderr is not None:
+        sys.stderr.write(pmm_agent_stderr)
+        sys.stderr.flush()
```

Comment:
I'm not sure why we need to handle stdout/stderr ourselves. Let it inherit file descriptors from the parent process.

Reply:
agree, removed.